internal/users/data/custom-password-hash-example.json

```diff
@@ -0,0 +1,96 @@
+[
+    {
+        "email": "antoinette@contoso.com",
```

Comment:
I think we could keep just one option here now that the template gets opened by default and users can edit it, so they wouldn't have to remove so much in here. Wdyt?

Reply:
I pulled these examples directly from the user import docs. I think each one is important. 
https://auth0.com/docs/manage-users/user-migration/bulk-user-import-database-schema-and-examples#examples